operator/hack/.checked-pr-template.md

```diff
@@ -0,0 +1,51 @@
+Thanks submitting your Operator. Please check below list before you create your Pull Request.
```

Comment:
Can you add this file to `operator/hack/.gitignore`, so that it does not get added by accident again?

Reply:
This is a file we always want to have in the repo.